great_expectations/cli/suite.py

```diff
@@ -529,6 +537,7 @@ def suite_edit(
         context=context,
         expectation_suite_name=expectation_suite,
         profile=False,
+        profiler_name=None,
```

Comment:
`suite edit` does not have the option of `--profile` but we need to adhere to the given signature. We could set a default of `profiler_name=None` but I thought I'd be a bit more explicit.